docs/src/pages/patterns/Header/Header.example.jsx

```diff
@@ -21,87 +13,55 @@ class HeaderExample extends React.PureComponent {
 
     this.state = {
       locale: 'en',
-      variation: 'logged-in',
-      inversed: true,
     };
   }
 
   renderHeaderToggles() {
-    const { locale, variation, inversed } = this.state;
-
-    const variations = variationChoices.map((choice) => ({
-      ...choice,
-      checked: choice.value === variation,
-    }));
+    const { locale } = this.state;
     const locales = localeChoices.map((choice) => ({
       ...choice,
       checked: choice.value === locale,
     }));
 
     return (
       <div className="ds-u-margin-top--4">
-        <ChoiceList
-          label="Change variation"
-          name="variation"
-          choices={variations}
-          onChange={(event) => this.setState({ variation: event.target.value })}
-          type="radio"
-        />
         <ChoiceList
           label="Change language"
           name="locale"
           choices={locales}
           onChange={(event) => this.setState({ locale: event.target.value })}
           type="radio"
         />
-
-        <fieldset className="ds-c-fieldset">
-          <legend className="ds-c-label">Change theme</legend>
-          <Choice
-            name="inversed"
-            value="inversed"
-            onChange={(event) =>
-              this.setState({ inversed: event.target.checked })
-            }
-            checked={inversed}
-            type="checkbox"
-          >
-            Inverse header
-          </Choice>
-        </fieldset>
       </div>
     );
   }
 
   render() {
-    const { locale, variation, inversed } = this.state;
-    const headerProps = {
-      home: {},
-      product: { subhead: 'Tax Tools', subpath: 'tax-tool/' },
-      'product-de': {
-        subhead: 'Verify your identity',
-        deConsumer: true,
-        deBrokerName: 'Acme Co.',
-      },
-      'logged-in': { loggedIn: true, firstName: 'Maximiliano-Longname' },
-      'logged-in-de': {
-        loggedIn: true,
-        deConsumer: true,
-        deBrokerName: 'Acme Co.',
-      },
-    };
-    let wrapperClassNames = 'ds-u-padding--1';
-    if (!inversed) {
-      wrapperClassNames += ' ds-base--inverse';
-    }
+    const { locale } = this.state;
+    const wrapperClassNames = 'ds-u-padding--1';
 
     return (
       // Add min-height so the options don't get cut off when switching to product-de
       <div style={{ minHeight: 679 }}>
+        <h6 className="preview__label">Header home</h6>
+        <Header initialLanguage={locale} />
+        <h6 className="preview__label">Header product</h6>
+        <Header subhead="Tax Tools" subpath="tax-tool/" initialLanguage={locale} />
+        <h6 className="preview__label">Header logged in</h6>
+        <Header loggedIn="true" firstName="Maximiliano-Longname" initialLanguage={locale} />
```

Comment:
I think this should be:
```suggestion
        <Header loggedIn={true} firstName="Maximiliano-Longname" initialLanguage={locale} />
```
Same for other instances of `loggedIn` below, as well as `deConsumer`.

Reply:
Updated, thanks @LouisFettet !